tests/shell-out/old.earth

```diff
@@ -1,104 +1,104 @@
 VERSION 0.6 # do not change to 0.7; this test is for old functionality
 
 test1:
-    FROM alpine:3.15
+    FROM alpine:3.18
```

Comment:
out of scope but we should probably move the image in this file and in other similar files to the base target

Reply:
we could... I don't feel strongly one way or the other, a tool like `find` + `sed` makes it easy enough to update these tests that I don't think we need to worry about the duplicated `FROM ...` lines.